jina/docker/hubapi.py

```diff
@@ -1,15 +1,70 @@
 import json
+import os
+import pkgutil
+from pkgutil import iter_modules
 from typing import Dict, Sequence, Any, Optional
 from urllib.error import HTTPError
 from urllib.parse import urlencode
 from urllib.request import Request, urlopen
 
 from pkg_resources import resource_stream
+from setuptools import find_packages
 
 from .helper import credentials_file
-from ..helper import yaml, colored
+from ..helper import colored, yaml
+from ..logging import default_logger
 from ..logging.profile import TimeContext
 
+_header_attrs = ['bold', 'underline']
+
+
+def _load_local_hub_manifest():
+    namespace = 'jina.hub'
+    try:
+        path = os.path.dirname(pkgutil.get_loader(namespace).path)
+    except AttributeError:
+        default_logger.warning('local Hub is not initialized, '
+                               'try "git submodule update --init" if you are in dev mode')
+        return {}
+
+    def add_hub():
+        m_yml = f'{info.module_finder.path}/manifest.yml'
+        if info.ispkg and os.path.exists(m_yml):
+            try:
+                with open(m_yml) as fp:
+                    m = yaml.load(fp)
+                    hub_images[m['name']] = m
+            except:
+                pass
```

Comment:
log exception here?

Reply:
on purpose, dont want to disturb users here, missing local hub is non-critical and can be fully suppressed